sphinx_gallery/backreferences.py

```diff
@@ -133,6 +133,8 @@ def scan_used_functions(example_file, gallery_conf):
     return backrefs
 
 
+# XXX This figure:: uses a forward slash even on Windows, but the op.join's
+# elsewhere will use backslashes...
```

Comment:
I think this kind of thing should be looked at indeed.

> even on Windows(I can't test it, don't own a windows machine,).

Now that we officially support Windows you probably want to have a setup where you can test locally on Windows. I personally have a Windows VM with VirtualBox. At the time you could download the Windows 10 ISO for free from the Microsoft website.